opentracing-impl-java8/src/test/java/io/opentracing/propagation/TestTextMapExtractorImpl.java

```diff
@@ -13,17 +13,16 @@
  */
 package io.opentracing.propagation;
 
+import io.opentracing.SpanContext;
+import io.opentracing.TestSpanContextImpl;
+
 import java.util.Iterator;
 import java.util.Map;
 
-import io.opentracing.propagation.TextMapReader;
-import io.opentracing.SpanContext;
-
-public class TestTextMapExtractorImpl implements Extractor {
-    public SpanContext extract(Object carrier) {
-        TextMapReader textMapCarrier = (TextMapReader)carrier;
```

Comment:
(admittedly this is just on the Tracer impl side which is far less important than the user code)